README.md

```diff
@@ -1,2 +1,166 @@
-# roadmap
-Master roadmap for Project Jupyter
+# Project Jupyter Roadmap
+
+The following is a very rough outline of the roadmap for Project Jupyter and IPython.
+Historically, we have published major releases every 6-9 months. With our project now
+split across many subprojects, we expect to release subprojects separately and more
+often.
+
+This roadmap only includes mainline subproject under the Jupyter
+(https://github.com/jupyter) and IPython (https://github.com/ipython) GitHub orgs. We
+also have a number of newer, experimental subprojects in the Jupyter Incubator
+(https://github.com/jupyter-incubator) organization.
+
+In this roadmap, we refer to the future major releases as follows:
+
+* Next (the next major release)
+* Next+1 (the major release after that)
+* Future (any release beyond the Next+1)
+
+This reflects the inherant uncertainty in releases that are more than about 1 year away. In
+some cases, if progress is made more quickly, features may be moved to earlier releases.
+
+## Documentation
+
+We are choosing to list documentation as a top-level section in our roadmap to acknowledge
```

Comment:
Perhaps cast this in a more positive, forward looking light.

We list documentation as a top-level section in our roadmap to acknowledge the importance of it for developer communication, scaling and maintenance of deployments, and early user adoption of new releases. The state of the existing documentation has not kept pace with the rapid development in our projects, and effort will be focused on improving it. Because our documentation...